soroban-test-wasms/wasm-workspace/loadgen/src/lib.rs

```diff
@@ -0,0 +1,49 @@
+#![no_std]
+use soroban_sdk::{contract, contractimpl, Bytes, Env, U256};
+
+#[contract]
+pub struct Contract;
+
+
+const EXTEND_LEDGER: u32 = 5_000_000;
+
+#[contractimpl]
+impl Contract {
+
+    // This will write num_entries of size size_kilo_bytes. The initial entry will have a key of
+    // starting_index, the next key starting_index + 1...
+    pub fn write(e: Env, starting_index: u32, num_entries: u32, size_kilo_bytes: u32) {
+        if num_entries == 0 || size_kilo_bytes == 0{
```

Comment:
nit: Could you please run `cargo fmt` on this? I suppose test Wasms are currently ignored by the make file formatting...